src/api/ledger/parse/utils.js

```diff
@@ -62,8 +62,8 @@ function parseOutcome(tx: Object): ?Object {
     fee: utils.common.dropsToXrp(tx.Fee),
     balanceChanges: balanceChanges,
     orderbookChanges: orderbookChanges,
-    ledgerVersion: tx.ledger_index,
-    indexInLedger: tx.meta.TransactionIndex
+    ledgerVersion: tx.ledger_index || 0,
```

Comment:
Thinking about this again makes me realize we probably shouldn't put 0 there. When are these fields not present? I believe it's when you get a transaction that is not yet validated right? In that case I think we want these fields to be not present.